tabs/mission_control.js

```diff
@@ -739,8 +737,8 @@ TABS.mission_control.initialize = function (callback) {
     }
 
     function saveMissionFile(filename) {
-        const fs = require('fs-extra');
-        const xml2js = require('xml2js');
+        const fs = require('fs');
```

Comment:
@mirko-it no problems with this package

Reply:
should be something like native, it is working even in the executable, while "fs-extra" was not.
added an error log for both packages if not found